coral/src/services/api-mocks/browser.ts

```diff
@@ -1,5 +1,9 @@
 import { setupWorker } from "msw";
 
-const worker = setupWorker();
+// eslint-disable-next-line
+let worker: any;
+if (typeof jest === "undefined") {
```

Comment:
Candid question because I don't understand: what was the reason for this addition? In which instance is `jest` `undefined` and why do we call `setupWorker` only when it is the case?

Reply:
Good question! Probably should have added a comment. 

This is a little hack to use the mocked API to show pagination in the app, without effecting tests. 
jest is `undefined` in the browser. Without that condition, `setupWorker` will be called when the code runs in tests and that will throw an error. This will not be committed in the end!